### PR TITLE
[plg_system_debug]Code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for plg_system_debug
### Testing Instructions

code review

### CODE REVIEW

File: plugins/system/debug/debug.xml [hunk 1]
```diff
@@ -163,35 +198,42 @@
 				/>
 			</fieldset>
 
-			<fieldset name="logging"
+			<fieldset
+				name="logging"
 				label="PLG_DEBUG_LOGGING_FIELDSET_LABEL">
```
Comment (JoomliC) [inline plugins/system/debug/debug.xml]: Wondering if ">" should be on a new line ? (as when Element is not empty...)

File: plugins/system/debug/debug.xml [hunk 2]
```diff
@@ -91,67 +107,86 @@
...
+					>
 					<option value="0">PLG_DEBUG_FIELD_LOG_CATEGORY_MODE_INCLUDE</option>
 					<option value="1">PLG_DEBUG_FIELD_LOG_CATEGORY_MODE_EXCLUDE</option>
 				</field>
 			</fieldset>
 
-			<fieldset name="language"
+			<fieldset
+				name="language"
 				label="PLG_DEBUG_LANGUAGE_FIELDSET_LABEL">
```
Comment (JoomliC) [inline plugins/system/debug/debug.xml]: Wondering if ">" should be on a new line ? (as when Element is not empty...)

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 177d4e908204658dfea49a4007f4bf1e5c9ace51<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10497">issues.joomla.org/joomla-cms/10497</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 177d4e908204658dfea49a4007f4bf1e5c9ace51<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10497">issues.joomla.org/joomla-cms/10497</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10497">issues.joomla.org/joomla-cms/10497</a>.</sub>

**Pull request merged**